config/initializers/cypress_rails.rb

```diff
@@ -7,8 +7,11 @@
 CypressRails.hooks.before_server_start do
   # Called once, before either the transaction or the server is started
   puts "Starting up server for end to end tests."
-  Rails.application.load_tasks
-  Rake::Task["db:seed:e2e"].invoke
+
+  if ENV["SKIP_SEED_DATA"].blank?
```

Comment:
Only seed data if we aren't skipping the seed data phase.

Reply:
I was using `unless ENV["SKIP_SEED_DATA"].present?` @djuber, but Rubocop asked me to go the if route instead.

Reply:
In case you missed my comment on Slack, we have a string refinement that adds a `to_boolean` method. 

```ruby
using StringToBoolean
unless ENV["SKIP_SEED_DATA"].to_boolean
```

But I think version you currently have is ok, we can keep that.

Reply:
I'll stick with what I have as we really only care about the existence of the environment variable `SKIP_SEED_DATA`.